src/vcProject.cpp

```diff
@@ -844,3 +848,66 @@ void vcProject_ExtractAttributionText(udProjectNode *pFolderNode, const char **p
 
   
 }
+
+void vcProject_UpdateProjectInformationDisplayTextures(vcState *pProgramState)
+{
+  const char *pInfo = nullptr;
+  if (udProjectNode_GetMetadataString(pProgramState->activeProject.pRoot, "information", &pInfo, "") == udE_Success)
+  {
+    for (vcTexture **ppTexture : pProgramState->projectInfoTextures.textures)
+    {
+      vcTexture_Destroy(ppTexture);
+      udFree(ppTexture);
+    }
+    for (const char *pStr : pProgramState->projectInfoTextures.infoStrings)
+      udFree(pStr);
+    pProgramState->projectInfoTextures.infoStrings.Clear();
+    pProgramState->projectInfoTextures.textures.Clear();
+
+    // Discover Images
+    int firstIndex = 0;
+    int startStrIndex = 0;
+    while (true)
+    {
+      size_t startImagePos = 0;
+      if (udStrchr(pInfo + firstIndex, "!", &startImagePos) == nullptr)
+        break;
+      startImagePos += firstIndex;
+      size_t closeBracketPos = 0;
+      if (udStrchr(pInfo + startImagePos + 1, "]", &closeBracketPos) == nullptr)
+        break;
+      closeBracketPos += startImagePos + 1;
+      size_t closeParenthesisPos = 0;
+      if (udStrchr(pInfo + closeBracketPos + 1, ")", &closeParenthesisPos) == nullptr)
+        break;
+      closeParenthesisPos += closeBracketPos + 1;
+        
+      size_t lastCharPos = closeParenthesisPos;
+      firstIndex = (int)(lastCharPos + 1);
+
+      if (pInfo[startImagePos + 1] == '[' && pInfo[closeBracketPos + 1] == '(')
+      {
+        // Text
+        size_t prevStrLen = startImagePos - startStrIndex;
+        const char *pSubStr = udStrndup(pInfo + startStrIndex, prevStrLen);
+        pProgramState->projectInfoTextures.infoStrings.PushBack(pSubStr);
+
+        // Texture Alt Text
+        const char *pAltStr = udStrndup(pInfo + startImagePos + 2, closeBracketPos - startImagePos - 2);
+        pProgramState->projectInfoTextures.textureAltStrings.PushBack(pAltStr);
+
+        // Texture
+        const char *pTexPath = udStrndup(pInfo + (closeBracketPos + 2), closeParenthesisPos - closeBracketPos - 2);
+        vcTexture **ppNewTexture = udAllocType(vcTexture*, 1, udAF_Zero);
```

Comment:
This is not a great way of doing this. If you keep the `textures` chunked array as `udChunkedArray<vcTexture*>` you can just push back with no parameters and it will return a `vcTexture **`. The function is marked as deprecated, but you can ignore this as we use it often and it's deprecation was not discussed properly, and breaks the pattern you want right here due to argument ambiguity with the other two overloads.